src/compiler/types.ts

```diff
@@ -1618,7 +1621,7 @@ namespace ts {
         closingElement: JsxClosingElement;
     }
 
-    /// Either the opening tag in a <Tag>...</Tag> pair, or the lone <Tag /> in a self-closing form
+    /// Either the opening tag in a <Tag>...</Tag> pair, the opening tag in a <>...</> pair, or the lone <Tag /> in a self-closing form

```

Comment:
This comment doesn't match our usage/terminology of `isJsxOpeningLikeElement`

Reply:
Fixed.